daffodil-runtime2/src/main/resources/examples/ex_nums.c

```diff
@@ -660,10 +559,10 @@ littleEndian_unparseSelf(const littleEndian *instance, UState *ustate)
 static void
 ex_nums_initSelf(ex_nums *instance)
 {
+    instance->_base.erd = &ex_nums_ERD;
     array_initSelf(&instance->array);
     bigEndian_initSelf(&instance->bigEndian);
     littleEndian_initSelf(&instance->littleEndian);
-    instance->_base.erd = &ex_nums__ERD;
 }
 
```

Comment:
What are these src/main/resources/examples files for? Are these examples of what the code generate will create? Are these for testing purposes? Perhaps they should be in src/test/resources?